### DISCUSSION TITLE
Can I set a custom size for the progress bar and increment it for progress?

### DISCUSSION THREAD

**Discussion by rew1nter** [charmbracelet/bubbles#298] [Q&A] [answered]
I'm trying to do something like [this](https://github.com/schollz/progressbar#basic-usage). Is this or something similar possible?

**Comment by fzdwx** [comment]
I have done something similar, you can refer to https://github.com/fzdwx/infinite/blob/main/_examples/progress/main.go

**Comment by rew1nter** [comment, marked as answer]
I figured out this is easily doable using the ViewAs() func. You just have to divide 1 by your total progress count and the result is what you increment for each progress.
